lib/libifconfig/libifconfig_carp.c

```diff
@@ -177,7 +177,7 @@ ifconfig_carp_set_info(ifconfig_handle_t *h, const char *name,
 	    &carpr->carpr_addr);
 	snl_add_msg_attr(&nw, CARP_NL_ADDR6, sizeof(carpr->carpr_addr6),
 	    &carpr->carpr_addr6);
-	snl_add_msg_attr_string(&nw, CARP_NL_KEY, carpr->carpr_key);
+	snl_add_msg_attr_string(&nw, CARP_NL_KEY, (caddr_t)carpr->carpr_key);
```

Comment:
That wants to be (const caddr_t).
I'll amend the commit.

Reply:
`const T`, where `T` is a typedef of `t *` is the same as `t * const`, i.e., a const pointer rather than a pointer to const. We have `c_caddr_t` for this purpose, but yeah it should just be `const char *`. We shouldn't be adding new uses of `caddr_t`.